docs/usage.md

```diff
@@ -128,11 +128,15 @@ Full list of features metadata are available on GEO platform pages.
 
 ## Contrasts file
 
+The contrasts file references the observations file to define groups of samples to compare. It can be provided in both CSV/TSV and YAML format using the parameters `--contrasts` or `--contrasts_yml`, respectively. Please note that only one of these parameters can be used at the same time.
```

Comment:
```suggestion
The contrasts file references the observations file to define groups of samples to compare. It can be provided in **either** CSV/TSV or YAML format using the parameters `--contrasts` or `--contrasts_yml`, respectively. 
```